src/app/api/post/createPost/index.ts

```diff
@@ -0,0 +1,41 @@
+// api/post/createPost/index.ts
+import baseUrl from "@/app/utils/baseUrl";
+import getToken from "@/app/utils/toke";
+import axios from "axios";
+import Swal from "sweetalert2";
+
+const createPost = async (postData: { title: string; description: string }) => {
```

Comment:
Same here, you can create a separate type or interface to set it to the variable, this because you can use the type once again.
IE:
interface CreatePost { title: string; description: string }

and, in the function, you can call it like this: 
async (postData: CreatePost)...